docs/changes.txt

```diff
@@ -150,6 +150,13 @@ All (GUI):
   in wxPGArrayEditorDialog.
 - Fix wxPropertyGrid issues with horizontal scrolling.
 - Add wxPG_DIALOG_TITLE wxPGProperty attribute.
+- Add support for creating a wxGraphicsPen with a gradient. The gradient will 
```

Comment:
Just FYI, I've actually stopped updating the change log directly to avoid merge conflicts and now keep the notes about the changes in git notes, see [this post](https://groups.google.com/d/msg/wx-dev/NVSfcnBygmk/SAQYuHTWDwAJ). But this can stay here, of course.

I would, however, try to keep it brief, as our change logs tend to be pretty long (yes, maybe the time between the releases has something to do with it), so I'd just leave
```
- Add support for creating a wxGraphicsPen with a gradient.
- Add support for applying a transformation matrix to a gradient.
```

Reply:
That is a good idea. I hadn't known about the git notes commands, I should take a closer look and consider that for wxPython too.